extensions/resteasy-classic/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/JaxRsSecurityConfig.java

```diff
@@ -1,22 +1,24 @@
 package io.quarkus.resteasy.runtime;
 
-import java.util.List;
-import java.util.Optional;
-
-import io.quarkus.runtime.annotations.ConfigItem;
 import io.quarkus.runtime.annotations.ConfigPhase;
 import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithName;
+
+import java.util.List;
+import java.util.Optional;
 
 /**
  * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
  */
-@ConfigRoot(name = "security.jaxrs", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
-public class JaxRsSecurityConfig {
+@ConfigRoot(phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
```

Comment:
Interestingly, we don't need this during the runtime, it could be moved to the deployment module and same change can be done in Quarkus REST. I don't think anyone has use case for this at runtime, it is too late for security...

Reply:
Ideally, let's adjust the config in follow up PRs.

I know it might look tempting to do the changes right away but it might make things confusing or harder to revert if we need to revert a specific adjustment.

So let's do the simple conversion and let's apply the cleanup afterwards.

Reply:
Sure, if interested, @IvanPetkov23 can provide a follow-up PR once this one is merged. Otherwise I'll take care of it when there is a time.